### PR TITLE
A slight modification of Python Server to introduce Social Wellness

### PR DESCRIPTION
**Pull request by edumorlom**
This includes Python changes to the Server Side to introduce Social Wellness statistical variables.

### CODE REVIEW

File: covid19-dashboard/calculate_data.py [hunk 1]
```diff
@@ -198,7 +198,8 @@ def _clean_dataset(date_to_value: DateToValueListType,
 
 def calculate_data(cumulative_stats: DateToValueDictType,
                    population: int,
-                   moving_average_chunk_size: int = 7
+                   moving_average_chunk_size: int = 7,
+                   clean_step_size: int = 2
```
Comment (beets) [inline covid19-dashboard/calculate_data.py:202]: missing doc for the new param?

File: covid19-dashboard/main.py [hunk 2]
```diff
@@ -33,33 +35,45 @@
...
-def county_data(geo_id: str):
+def data(geo_id: str, dashboard_id: str):
     """
-    Returns any placeType's datta.
+    Returns any placeType's data.
     NOTE: for return type documentation, please see README.md's APIs section.
     :return: geo_id->{**key_to_timeseries}.
     """
+    dashboard_id = dashboard_id or "covid19"
+    print(dashboard_id)
```
Comment (beets) [inline covid19-dashboard/main.py:59]: remove debugging statements

File: covid19-dashboard/main.py [hunk 3]
```diff
@@ -33,33 +35,45 @@
...
+
+    # How many days should we perform the moving average for?
+    # 0 == None
+    moving_averages_chunks = {"covid19": 7, "socialWellness": 0}
```
Comment (beets) [inline covid19-dashboard/main.py:63]: maybe add a todo for these options to be config driven

**Review by beets — COMMENTED** [review]
nice, just a few small comments

File: covid19-dashboard/calculate_data.py (hunk 1 above)
Comment (edumorlom) [reply covid19-dashboard/calculate_data.py:202]: Fixed.

File: covid19-dashboard/calculate_data.py [hunk 4]
```diff
@@ -233,8 +234,11 @@ def calculate_data(cumulative_stats: DateToValueDictType,
     difference_list = _difference(date_to_value_list, 1)
 
     # Calculate the time-series moving average.
-    moving_averages_list = _moving_average(difference_list,
-                                           moving_average_chunk_size)
+    # TODO: fix this
```
[the remark this answers is not in the document]
Comment (edumorlom) [reply covid19-dashboard/calculate_data.py:237]: Sorry, that was actually fixed. I just forgot to remove the #TODO. It was never meant to be commited.

File: covid19-dashboard/main.py (hunk 2 above)
Comment (edumorlom) [reply covid19-dashboard/main.py:59]: fixed.

File: covid19-dashboard/main.py (hunk 3 above)
Comment (edumorlom) [reply covid19-dashboard/main.py:63]: Done.

**Comment by edumorlom** [comment]
Ready for re-review! I am working on a better README. Will do a separate PR.
Thanks!

File: covid19-dashboard/main.py [hunk 5]
```diff
@@ -46,6 +46,8 @@
 GeoIdToStatsType = Dict[str, Dict[str, int]]
 PlaceToInfoType = Dict[str, Dict[str, str]]
 
+print("HI WORLD")
```
Comment (beets) [inline covid19-dashboard/main.py:49]: remove this?

**Review by beets — COMMENTED** [review]
ah, sorry for sitting on this for so long! in the meantime, a debug statement snuck in..

File: covid19-dashboard/main.py (hunk 5 above)
Comment (edumorlom) [inline covid19-dashboard/main.py:49]:
```suggestion
```

Comment (edumorlom) [reply covid19-dashboard/main.py]: fixed

**Comment by edumorlom** [comment]
Whoops, I removed the print statement. Thank you!

**Review by beets — APPROVED** [review]

**Pull request merged**